net/smartdns/Makefile

```diff
@@ -7,12 +7,12 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=smartdns
 PKG_VERSION:=1.2022.38
```

Comment:
It is, after all, a source c code change?
Shouldn't we then update the PKG_VERSION instead?
PKG_RELEASE should be update if changes where made in the openwrt Makefile or in files by openwrt provided.

Reply:
change to PKG_VERSION: 1.2022.38.1, PKG_RELEASE:=1

Reply:
Please update the PKG_MIRROR_HASH https://github.com/openwrt/packages/actions/runs/3234170838/jobs/5296948001#step:5:3433

Reply:
done.